docs/next.config.js

```diff
@@ -19,6 +19,37 @@ module.exports = withNextPluginPreval({
     ignoreBuildErrors: true,
   },
 
+  async headers() {
+    return [
+      {
+        // Apply these headers to all routes in your application.
+        source: '/(.*)',
+        headers: [
```

Comment:
as `yaml` is already a dependency of some packages, can we read these values from the `customHttp.yml` file?

Reply:
Is this blocking, since this only relates to the dev build? Would prefer to do as follow up item since this is a security issue

Reply:
sure! np